docs/tutorials/intro_tutorial.ipynb

```diff
@@ -33,21 +33,27 @@
     " (You can use the table of contents button on the left side of the interface to skip to any specific part)\n",
     "\n",
     "\n",
-    "Users can use Google Colab (Please ensure you run the Colab dependency import cell) \n",
+    "Users can use Google Colab\\* (Please ensure you run the Colab dependency import cell- below) \n",
     "\n",
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/projectmesa/mesa-geo/blob/main/docs/tutorials/intro_tutorial.ipynb)\n"
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/projectmesa/mesa-geo/blob/main/docs/tutorials/intro_tutorial.ipynb)\n",
+    "\n",
+    "\\*Based on a recent Google Colab update currently, Solara visuals are not rendering. However, the link still provides a easy way to download the jupyter file. \n",
```

Comment:
Would be informative to add the link to the issue https://github.com/widgetti/solara/issues/323.